google/cloud/spanner/samples/samples.cc

```diff
@@ -98,6 +100,20 @@ void AddColumn(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void QueryWithStruct(std::vector<std::string> const& argv) {
+  // TODO(tmatsuo): Add querying part once data client is ready.
```

Comment:
We try to put github issues in our TODOs rather than usernames. Here, I think there's already an issue for this, so I can you can write `TODO(#191)`

Reply:
I did it with #188